inc/sairouterintf.h

```diff
@@ -45,7 +45,13 @@ typedef enum _sai_router_interface_type_t
     SAI_ROUTER_INTERFACE_TYPE_VLAN,
 
     /** Loopback Router Interface Type */
-    SAI_ROUTER_INTERFACE_TYPE_LOOPBACK
+    SAI_ROUTER_INTERFACE_TYPE_LOOPBACK,
+
+    /** Sub port Router Interface Type */
+    SAI_ROUTER_INTERFACE_TYPE_SUB_PORT,
+
+    /** .1D Bridge Router Interface Type */
+    SAI_ROUTER_INTERFACE_TYPE_BRIDGE
```

Comment:
Is it possible to create a RIF on a sub port and then create a bridge port using the same and then create a RIF on the .1D bridge router interface?